### PR TITLE
Awesome-books Navigation

### PR DESCRIPTION
**Pull request by 1GORDON**
## In this milestone we achieved the following
- Added Navigation to Awesome books SPA

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,14 +11,29 @@
   <title>Awesome books</title>
```
Comment (smcommits) [inline index.html:11]: - [ ] **Please add a footer to your website as reflected in the wireframe.**

The wireframe provided in the [project requirement repo](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements) contains a footer that is missing on your website. Please add a footer.  Being able to follow a design guideline is a very important skill for any developer.

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @1GORDON, 

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. 
### To make this project even better, there are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️
_Please check the inline comments below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (1GORDON) [reply index.html:11]: Thank you so much for the review. I had forgotten the footer element. My bad

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @1GORDON and @zieeco , 

I can see you added a proper footer 👍 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 1GORDON** [comment]
Thank you.

**Pull request merged**